src/main/java/com/kvlabs/filemock/FileToMock.java

```diff
@@ -0,0 +1,55 @@
+package com.kvlabs.filemock;
+
+import com.kvlabs.filemock.mapper.Mapper;
+import com.kvlabs.filemock.mapper.Deserializer;
+import com.kvlabs.filemock.mapper.DeserializerFactory;
+
+/**
+ *
+ * @author kanchana-prasanth
+ */
+public class FileToMock {
+
+    /**
+     * Creates mock object of given class with the JSON data provided in the
+     * path.
+     *
+     * @param <T> as the class type
+     * @param filePath as the path to the mock file
+     * @param classToMock class or interface to mock
+     * @return mock object
+     */
+    public static <T> T mock(String filePath, Class<T> classToMock) {
```

Comment:
possible conflict with [mockito](http://site.mockito.org/) if end user is using both.